src/routes/+layout.svelte

```diff
@@ -48,15 +48,34 @@
     import { derived, writable } from 'svelte/store';
     import { navigating, page } from '$app/stores';
     import { onMount } from 'svelte';
+    import { sdk, loggedIn } from '$lib/utils/console';
 
     function applyTheme(theme: Theme) {
         const resolvedTheme = theme === 'system' ? getSystemTheme() : theme;
         const className = `theme-${resolvedTheme}`;
         document.body.classList.remove('theme-dark', 'theme-light');
         document.body.classList.add(className);
     }
+    let getAccount = async () => {
+      loggedIn.set(
+        localStorage.getItem('appwrite:loggedIn') !== null
```

Comment:
is this a convention of how we store things in localstorage? do we have this documented?